spec/requests/conversations/show_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GET /conversations/:id', type: :request do
+  let(:conversation) { create :conversation }
+
+  subject do
+    get "/conversations/#{conversation.id}", headers: headers
```

Comment:
you can use the path method that you can check with `rails routes` instead of creating the url manually

Reply:
I feel like this way I'm also testing the routes, the same way the client would actually call these endpoints. But I can change them, what do you think?